Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -120,7 +195,14 @@ struct SelectTracks {
         if ((status_prong & (1 << 1)) && (abs(dca[0]) < dcatoprimxymin_3prong_ptdep || abs(dca[0]) > dcatoprimxymax_3prong)) {
           status_prong = status_prong & ~(1 << 1);
         }
+        if ((status_prong & (1 << 2)) && (abs(dca[0]) < d_dcaToPrimXYMinBach_ptDep || abs(dca[0]) > d_dcaToPrimXYMaxBach)) {
+          MY_DEBUG_MSG(isProtonFromLc,
+                       LOG(INFO) << "proton " << protonLabel << " did not pass DCA cut";
+                       LOG(INFO) << "dca[0] = " << dca[0] << " (lower cut " << d_dcaToPrimXYMinBach_ptDep << ", upper cut " << d_dcaToPrimXYMaxBach << ")";);
+          status_prong = status_prong & ~(1 << 2);
+        }
```

Comment:
This part needs to be reworked with the array cuts.

Reply:
Yes, for later